doc/3.Algorithms.md

```diff
@@ -322,13 +322,13 @@ Assertion 3.6.8:
 
 Assertion 3.6.9:
     SpdmMessage.BaseAsymSel only has one bit at most.
-    if (CHAL_CAP == 1 || MEAS_CAP == 2) then (BaseAsymSel == one of {TPM_ALG_RSASSA_2048, TPM_ALG_RSAPSS_2048, TPM_ALG_RSASSA_3072, TPM_ALG_RSAPSS_3072, TPM_ALG_ECDSA_ECC_NIST_P256, TPM_ALG_RSASSA_4096, TPM_ALG_RSAPSS_4096, TPM_ALG_ECDSA_ECC_NIST_P384, TPM_ALG_ECDSA_ECC_NIST_P521, TPM_ALG_SM2_ECC_SM2_P256, EdDSA_ed25519, EdDSA_ed448})
-    if (CHAL_CAP == 0 && MEAS_CAP != 2) then (BaseAsymSel == 0)
+    if (CERT_CAP == 1 || CHAL_CAP == 1 || MEAS_CAP == 2 || KEY_EX_CAP == 1) then (BaseAsymSel == one of {TPM_ALG_RSASSA_2048, TPM_ALG_RSAPSS_2048, TPM_ALG_RSASSA_3072, TPM_ALG_RSAPSS_3072, TPM_ALG_ECDSA_ECC_NIST_P256, TPM_ALG_RSASSA_4096, TPM_ALG_RSAPSS_4096, TPM_ALG_ECDSA_ECC_NIST_P384, TPM_ALG_ECDSA_ECC_NIST_P521, TPM_ALG_SM2_ECC_SM2_P256, EdDSA_ed25519, EdDSA_ed448})

```

Comment:
This also includes `PUB_KEY_ID_CAP`.

Reply:
I dont think we need PUB_KEY_ID_CAP. Current CAP is enough for all use cases.
We dont expect a device having PUB_KEY_ID_CAP=1, while all rests are 0.